src/schema/type/DefinedTermSet.graphql

```diff
@@ -0,0 +1,197 @@
+"https://pending.schema.org/DefinedTermSet"
+type DefinedTermSet implements CreativeWorkInterface & ThingInterface {
+    ### Metadata properties ###
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    "http://purl.org/dc/elements/1.1/contributor,https://schema.org/contributor"
+    contributor: String
+    "http://purl.org/dc/elements/1.1/coverage"
+    coverage: String
+    "http://purl.org/dc/elements/1.1/creator"
+    creator: String!
+    "http://purl.org/dc/elements/1.1/date"
+    date: _Neo4jDate
+    "http://purl.org/dc/elements/1.1/description,https://schema.org/description"
+    description: String
+    "https://schema.org/disambiguatingDescription"
+    disambiguatingDescription: String
+    "http://purl.org/dc/elements/1.1/format"
+    format: String
+    "http://purl.org/dc/elements/1.1/language"
+    language: AvailableLanguage
+    "http://purl.org/dc/elements/1.1/publisher"
+    publisher: String
+    "http://purl.org/dc/elements/1.1/relation"
+    relation: String
+    "http://purl.org/dc/elements/1.1/rights"
+    rights: String
+    "http://purl.org/dc/elements/1.1/source"
+    source: String
+    "http://purl.org/dc/elements/1.1/subject"
+    subject: String
+    "http://purl.org/dc/elements/1.1/title"
+    title: String
+    "http://purl.org/dc/elements/1.1/type,https://www.w3.org/2000/01/rdf-schema#type"
+    type: String
+    "http://purl.org/dc/elements/1.1/date"
+    created: String
+    "http://purl.org/dc/elements/1.1/date"
+    updated: String
+    #################################
+    ### ThingInterface properties ###
+    "https://schema.org/additionalType"
+    additionalType: [String]
+    "https://schema.org/additionalProperty"
+    additionalProperty: [PropertyValue] @relation(name: "ADDITIONAL_PROPERTY", direction: "OUT")
+    "https://schema.org/alternateName"
+    alternateName: String
+    "https://schema.org/image"
+    image: URL
+    "https://schema.org/mainEntityOfPage"
+    mainEntityOfPage: [CreativeWorkInterface] @relation(name: "MAIN_ENTITY_OF_PAGE", direction: "OUT")
+    "https://schema.org/name"
+    name: String!
+    "https://schema.org/potentialAction"
+    potentialAction: [ActionInterface] @relation(name: "POTENTIAL_ACTION", direction: "OUT")
+    "https://schema.org/sameAs"
+    sameAs: URL
+    "https://schema.org/subjectOf"
+    subjectOf: [CreativeWorkInterface] @relation(name: "SUBJECT_OF", direction: "OUT")
+    "https://schema.org/url"
+    url: URL
+    ###############################
```

Comment:
a DefinedTermSet is a CreativeWork according to https://schema.org/DefinedTermSet, so we need these additional fields. Same question above about DefinedTerm/Thing applies here. Is this strictly necessary?
If we find that we never need to link a DefinedTermSet to a CreativeWork field on another Type perhaps we don't need this?

Reply:
Yes, `DefinedTermSet` should actually implement `ThingInterface` and `CreativeWorkInterface`. DigitalDocument does the same thing: https://github.com/trompamusic/ce-api/blob/47707dd7a0d928a4a66457dfddf283585058620b/src/schema/type/DigitalDocument.graphql#L2

Reply:
OK. I've done this here, and you're right that it's defined like this in the schema. We'll leave it as-is for now.